docs/usage/validation_errors.md

```diff
@@ -32,10 +32,12 @@ except ValidationError as exc_info:
 
 ## `bool`
 
+This error is raised when the value type is not valid for a Boolean field.
+
 
 ## `bool_parsing`
 
-This error is raised when the value is a string that is not valid for coercion to a boolean.
+This error is raised when the value is a string that is not valid for coercion to a Boolean.
```

Comment:
```suggestion
This error is raised when the value is a string that is not valid for coercion to a boolean.
```

I recognize it's subjective, but I don't think we need to capitalize the B in Boolean; for reference, the `mypy` docs don't: https://mypy.readthedocs.io/en/stable/builtin_types.html.

If you prefer to capitalize anyway, you can ignore this.